pkg/kn/commands/service/delete.go

```diff
@@ -77,18 +78,22 @@ func NewServiceDeleteCommand(p *commands.KnParams) *cobra.Command {
 				}
 			}
 
+			errs := []string{}
 			for _, name := range args {
 				timeout := time.Duration(0)
 				if waitFlags.Wait {
 					timeout = time.Duration(waitFlags.TimeoutInSeconds) * time.Second
 				}
 				err = client.DeleteService(name, timeout)
 				if err != nil {
-					fmt.Fprintf(cmd.OutOrStdout(), "%s.\n", err)
+					errs = append(errs, err.Error())
```

Comment:
I can comment this line and all tests still pass. This tells more there are no new tests for this change. Please add

Reply:
@maximilien Added a test case. Now it should fail if the changes are reverted.